persistence/genesis.go

```diff
@@ -321,3 +321,30 @@ func (p PostgresContext) GetAllFishermen(height int64) (f []modules.Actor, err e
 	}
 	return
 }
+
+// IMPROVE: This is a proof of concept. Ideally we should have a single query that returns all actors.
+func (p PostgresContext) GetAllStakedActors(height int64) (actors []modules.Actor, err error) {
```

Comment:
I don't think `genesis.go` is the right place for this. Can you creator a new `actor.go` file?

Reply:
I agree with you but all the similar counterparts are here as well... I'll move all of them then.

On that note, I was about to start tackling #149 

```
// TODO(pocket/issues/149): All of the functions below following a structure similar to `GetAll<Actor>`
//	can easily be refactored and condensed into a single function using a generic type or a common
//      interface.
``` 
but then I remembered my promises from earlier today in planning. I might come back at it for vengeance later...

Reply:
Can you update that comment (since #149) was already resolved and just remove the ticket number for a plain TODO?

Reply:
I have created a new issue and referenced that so that it doesn't get lost in the `void`